dmHelper/dmHelper.py

```diff
@@ -23,9 +27,25 @@ def __init__(self, bot):
         self.bot = bot
         self.message_queue : list = []
         self.priority_message_queue : list = []
+        self.errored_message_queue : list = [] # used to store DMs that were unable to be delivered
         self.actively_sending = False
         # self.task = asyncio.create_task(self.process_dm_queues())  # TODO: protect queue from bot crashes -- json?
     
+    @commands.Cog.listener('on_message')
+    async def _message_listener(self, message: Discord.Message)
```

Comment:
```suggestion
    async def _message_listener(self, message: discord.Message):
```
We should also use message type to only listen to DMs
![image](https://user-images.githubusercontent.com/16944592/184258218-b7face37-2e21-40a3-8011-68a167e6476d.png)